src/chunk.js

```diff
@@ -43,18 +44,37 @@ const resolved = Promise.resolve();
  * there is time.
  *
  * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
- * @param {function()} fn Function that will be called as a "chunk".
+ * @param {function(?IdleDeadline)} fn
  */
-export function chunk(nodeOrAmpDoc, fn) {
+export function startupChunk(nodeOrAmpDoc, fn) {
```

Comment:
I don't understand why `startupChunk` implies "deactivate-able" while normal `chunk` is always "active".